### ISSUE TITLE
Unable to <alt> keybindings that match top level menus

### ISSUE THREAD

**Issue by mschuckmann** [Microsoft/vscode#24514]
- VSCode Version: Code 1.11.1 (d9484d12b38879b7f4cdd1150efeb2fd2c1fbf39, 2017-04-06T13:55:56.395Z)
- OS Version: Linux x64 4.4.0-72-generic
- Extensions: 

|Extension|Author|Version|
|---|---|---|
|markdown-toc|AlanWalk|1.5.1|
|numbered-bookmarks|alefragnani|0.7.0|
|project-manager|alefragnani|0.13.5|
|code-gnu-global|austin|0.2.1|
|togglehs|bbenoist|0.1.2|
|vscode-markdownlint|DavidAnson|0.7.1|
|jshint|dbaeumer|0.10.15|
|githistory|donjayamanne|0.2.0|
|python|donjayamanne|0.6.0|
|gitlens|eamodio|3.3.3|
|EditorConfig|EditorConfig|0.6.5|
|git-project-manager|felipecaputo|1.2.1|
|Angular2|johnpapa|2.2.3|
|vscode-clang|mitaki28|0.2.2|
|HTMLHint|mkaufman|0.3.3|
|prettify-json|mohsen1|0.0.3|
|cpptools|ms-vscode|0.10.5|
|Spell|seanmcbreen|0.9.1|
|code-settings-sync|Shan|2.6.2|
|cmake|twxs|0.0.17|
|cmake-tools|vector-of-bool|0.9.4|
|clang-format|xaver|1.2.1|;

---

I am unable to successfully use <alt> based keybindings that match the top level menus. 

I have the following custom key shortcuts
alt + D = delete line
alt + G = go to line

However when I press the key sequences the corresponding menu bar menus are shown instead of my bound shortcut actions, e.g.
alt + d = Debug menu
alt + g = Go menu

I have tried setting "window.menuBarVisibility": "hidden" but that appears to have no effect. 
I have had some success disabling the setting "Key to show the menu bar while pressed"  under Desktop > Ubuntu Unity Plugin in the "CompizConfig Settings Manager" application.  With this setting set to Disabled my alt key binding work in VSCode when it is initially launched but the behavior reverts after  to another application and then back to VSCode.

Steps to Reproduce:

1.
2.

**Comment by bpasero** [comment]
@mschuckmann can you still reproduce?

**Comment by mschuckmann** [comment]
@bpasero no I've been using alt key combos for sometime now so I'm good.
